cpp/dolfinx/common/TimeLogger.cpp

```diff
@@ -7,6 +7,7 @@
 #include "TimeLogger.h"
 #include <dolfinx/common/MPI.h>
 #include <dolfinx/common/log.h>
+#include <iostream>
```

Comment:
Is this include needed?

Reply:
Yes, because `std::cout` is used in the file.